gulpfile.js

```diff
@@ -7,10 +7,10 @@ var requireDir = require('require-dir');
 var path = require('path');
 var debug = require('gulp-debug');
 
-var iusPath = path.join(process.cwd(), 'node_modules/gulp-ius-web');
+var iusPath = path.join(process.cwd(), 'node_modules/gulp-benefits-web/node_modules/gulp-ius-web');
 var ius = requireDir(iusPath);
 
-var appSettingsPath = path.join(process.cwd(), 'node_modules/gulp-appsettings');
+var appSettingsPath = path.join(process.cwd(), 'node_modules/gulp-benefits-web/node_modules/gulp-appsettings');
```

Comment:
I am not sure whether we want to nest the node_modules, they were already being nested in the current code... with the current code, the path does need to be nested, because this gulp file gets called by another... project... gulpfile, which sets the working directory the location of the initial gulpfile... ie project root. Since we are now nesting node_modules, the path needs to be from the project root, not from the location of this gulpfile... my npm version is 2.15.9